src/core/stdc/signal.d

```diff
@@ -14,13 +14,14 @@ module core.stdc.signal;
 
 extern (C):
 @system:
-nothrow:
-@nogc:
 
 // this should be volatile
-alias int sig_atomic_t;
+alias sig_atomic_t = int;
+
+private alias sigfn_t = void function(int);
 
-private alias void function(int) sigfn_t;
```

Comment:
This shouldn't be a private alias to begin with. This type is visible via the public API.